_integration-test/ensure-sentry-admin-works.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```

Comment:
I don't think we should have two instances of backup in integration tests if it takes too long. It already takes over 25 minutes to run integration tests and this adds to that overhead

Reply:
Removed this 2nd backup test, and modified the original to use `sentry-admin.sh` instead. We'll modify the docs once this PR lands.